WalletWasabi/WabiSabi/Backend/DoSPrevention/Prison.cs

```diff
@@ -90,9 +88,7 @@ TimeSpan CalculatePunishment(Offender offender, RoundDisruption disruption)
 				});
 
 			var repetitions = offenderHistory.Count;
-			var repetitionFactor = CoinJoinIdStore.Contains(offender.OutPoint.Hash)
-				? repetitions                   // Linear punishment
-				: Math.Pow(2, repetitions - 1); // Exponential punishment
+			var repetitionFactor = Math.Pow(1.3, repetitions - 1); // Exponential punishment
```

Comment:
So here the narrative is that we now cannot know whether or not it's a fresh input, so we always punish exponentially, but therefore reduced the exponential factor?